cves/kernel/CVE-2020-12654.yml

```diff
@@ -393,37 +433,43 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
+    applies: false
```

Comment:
possibly distrust input? Given that not checking the passed in argument for length caused the system to fail when calling a method on the particular arg.